packages/ui/src/molecules/Modal/Modal.tsx

```diff
@@ -31,6 +31,14 @@ export interface ModalProps extends ModalContentProps {
    * Controls whether or not the dialog is open.
    */
   isOpen: boolean
+  /**
+   * ID to identify overlay
+   */
+  overlayId?: string
+  /**
+   * Returns the value of overlay's class attribute.
+   */
+  className?: string
```

Comment:
I think we should rename this to `overlayClassName` because there might already exist a `className` this is being sent to `ModalContent` inside `{...otherProps}`, so we avoid this confusion!